app/Controllers/importExportController.php

```diff
@@ -28,11 +28,16 @@ public function firstAction() {
 	/**
 	 * This action displays the main page for import / export system.
 	 */
-	public function indexAction() {
+	public function indexAction(): void {
 		$this->view->feeds = $this->feedDAO->listFeeds();
 		FreshRSS_View::prependTitle(_t('sub.import_export.title') . ' · ');
 	}
 
+	/**
+	 * @param $size_str
+	 * @return float|int
```

Comment:
We should probably always return float

Reply:
after retesting, I confirm that we can have integers and floats as results